src/textual/css/stylesheet.py

```diff
@@ -409,6 +415,12 @@ def apply(
         # Collect the rules defined in the stylesheet
         node._has_hover_style = False
         node._has_focus_within = False
+
+        # Rules that may have an initial value
```

Comment:
This comment isn't clear because the English word “initial” and the special CSS value `initial` clash.
I've had to reread these comments and the code below multiple times and each time my understanding of what we mean changes.

I suggest a slight rewording for the future person reading this code:

```suggestion
        # Rules that may be set to the special value `initial`
```

Reply:
I'm not sure how they clash. They pretty much mean the same thing. Just one is a token. But I don't mind rewording.